source/core/annotations/ut_trigger_check.pks

```diff
@@ -0,0 +1,33 @@
+create or replace package ut_trigger_check authid definer is
+  /*
+  utPLSQL - Version 3
+  Copyright 2016 - 2018 utPLSQL Project
+
+  Licensed under the Apache License, Version 2.0 (the "License"):
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+  */
+
+  gc_check_object_name constant varchar2(128) := 'UT3_TRIGGER_ALIVE';
```

Comment:
Would that not be better to define in utils for example and refer to it via package constant in trigger and this package ? Its only called twice but one less line to maintain ?

Reply:
This needs to be removed as no longer used - thanks